CHANGELOG.md

```diff
@@ -10,33 +10,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- *React:* Fixed an issue with registering modules for React wrapper [#9140](https://github.com/handsontable/handsontable/issues/9140)
+### Added
+- Added `ShortcutManager`, a new API for customizing keyboard shortcuts. [#8942](https://github.com/handsontable/handsontable/issues/8942)
+- Added support for RTL languages, by introducing a new configuration option: `layoutDirection`. [#8760](https://github.com/handsontable/handsontable/issues/8760)
+- Added an Arabic translation. [#9208](https://github.com/handsontable/handsontable/issues/9208)
+- Added a new configuration option: `fixedColumnsStart`. [#8760](https://github.com/handsontable/handsontable/issues/8760)
 
 ### Changed
-- **Breaking change**: Changed the way `updateSettings` handles data by introducing `updateData` instead of `loadData` in the majority of cases. [#7263](https://github.com/handsontable/handsontable/issues/7263)
-- **Breaking change**: Changed the way the plugins' `updatePlugin` method reacts to `updateSettings` calls. [#9021](https://github.com/handsontable/handsontable/issues/9021)
-- Changed the Code Examples Deployment workflow to make a dry-run on feature branches (without the actual deployment) [#8980](https://github.com/handsontable/handsontable/issues/8980)
+- **Breaking change**: Changed the way `updateSettings()` handles data, by introducing `updateData()` instead of `loadData()` in most cases. [#7263](https://github.com/handsontable/handsontable/issues/7263)
+- **Breaking change**: Changed the way any plugin's `updatePlugin()` method reacts to `updateSettings()` calls. [#9021](https://github.com/handsontable/handsontable/issues/9021)
+- **Breaking change**: Changed the order of execution for two hooks: now, `beforeKeyDown` is properly fired before `afterDocumentKeyDown`. [#6236](https://github.com/handsontable/handsontable/issues/6236)
 
-### Added
-- **Breaking change**: Introducing the ShortcutManager - a new API for handling shortcuts. [#8942](https://github.com/handsontable/handsontable/issues/8942)
-- Added a new configuration option: `fixedColumnsStart` [#8760](https://github.com/handsontable/handsontable/issues/8760)
-- Add Arabic Language [#9208](https://github.com/handsontable/handsontable/issues/9208)
+### Fixed
+- Fixed an issue where cell filtering used formula results instead of source values. [#5455](https://github.com/handsontable/handsontable/issues/5455)
+- Fixed an issue where the Handsontable documentation didn't build locally on Windows. [#8677](https://github.com/handsontable/handsontable/issues/8677)
+- Fixed a wrong TypeScript definition in the `BasePlugin` class. [#9175](https://github.com/handsontable/handsontable/issues/9175)
+- *React, Vue 2, Vue 3:* Fixed an issue with registering modules for the React, Vue 2, and Vue 3 wrappers. [#9140](https://github.com/handsontable/handsontable/issues/9140)
```

Comment:
According to the comment https://github.com/handsontable/handsontable/issues/8372#issuecomment-1048665298, Handsontable 12.0.0 fixes some bugs missed in this RN:

1. #7567
1. Some linked problems of #7263
   - For example https://github.com/handsontable/handsontable/issues/6552
   - @jansiegel please provide the full list as you were the fixer
1. #7165
1. Cases 1-5 listed in https://github.com/handsontable/handsontable/issues/8372 (remaining Case 6 should be moved to a new issue)

Reply:
@warpech done in [3e9465e](https://github.com/handsontable/handsontable/pull/9271/commits/3e9465e057588ad6b43eafc0905601dae9b474fc), except for no. 2 above

Reply:
@warpech @kirszenbaum 

2. I made the list here: https://github.com/handsontable/handsontable/issues/8372#issuecomment-1048665298

Reply:
Ok, so it looks like we have it all:

<img width="756" alt="2" src="https://user-images.githubusercontent.com/29247866/157621476-f78c742a-5506-41ac-8a7a-d8f541c455a8.png">

Reply:
Yes, I confirm this looks complete now. Thanks guys!